ntbs-service/Startup.cs

```diff
@@ -459,6 +458,7 @@ private void AddAuditService(IServiceCollection services, string auditDbConnecti
         {
             if (Configuration.GetValue<bool>(Constants.AuditEnabledConfigValue))
             {
+                Audit.Core.Configuration.JsonSettings.MaxDepth = 1;
```

Comment:
It might be worth adding a comment here saying that we only want to audit the simple properties of an object.

Reply:
We are setting it to 1 here, but 0 in the EFAuditExtensions class - which actually sets it to the default value of 64. If I set it there to 1 I get the same cycle errors as yesterday.